src/Analysis/Ast/Impl/Types/ArgumentSet.cs

```diff
@@ -38,7 +39,7 @@ internal sealed class ArgumentSet : IArgumentSet {
         private readonly DictArg _dictArgument;
         private bool _evaluated;
 
-        public static IArgumentSet Empty = new ArgumentSet();
+        public static IArgumentSet EmptyNoContext = new ArgumentSet();
```

Comment:
Perhaps just `WithoutContext`, such that it's used as `ArgumentSet.WithoutContext`.

Reply:
Done!